backend/fleet_management/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class FleetManagementConfig(AppConfig):
     name = 'fleet_management'
+
+    def ready(self):
+        from .signals import *
```

Comment:
It's a bad practice to import outside `imports section` on the top of the file. and please don't import `*`.
You may go with `from . import signals` or `from .signals import What, You, Need`

Reply:
https://docs.djangoproject.com/pl/2.1/topics/signals/
>  If you’re using the receiver() decorator, simply import the signals submodule inside ready().

Reply:
nice, thx